rename_images.sh

```
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Script to rename WhatsApp images to more descriptive names
+# This script will be deleted after use
+
+cd "$(dirname "$0")/public/images"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)